gms/api/src/main/pegasus/com/linkedin/glossary/GlossaryNodeKey.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.glossary
+
+import com.linkedin.common.GlossaryNodeUrn
+
+/**
+ * Key for GlossaryNode resource
+ */
+record GlossaryNodeKey {
```

Comment:
similarly, why is the NodeKey structured differently than the NodeUrn?

Reply:
will be removing `glossaryNodePath` [same as above]

Reply:
done